data-prepper-plugins/rds-source/src/main/java/org/opensearch/dataprepper/plugins/source/rds/model/DbTableMetadata.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.source.rds.model;
+
+import java.util.Map;
+
+public class DbTableMetadata {
```

Comment:
It's called DbTableMetadata, yet it doesn't contain metadata about the table, like table name and database (schema) name. Wondering how this model will be used. And why do you need DbMetadata in this model?

Reply:
Ok. I got most of it from description. When do we need DbMetadata stored in coordination store?

Reply:
I will have following PRs. This will be stored in global state. It will be used in BinLogListener and ExportWorker.